lib/google/gax/protobuf.rb

```diff
@@ -0,0 +1,169 @@
+# Copyright 2019, Google Inc.
```

Comment:
`Google LLC` please

Reply:
Should I change the company attribution on all files? Should we also change the 1.x branch?

Reply:
I'm going to spin this out into its own issue. #187